core/src/main/java/com/microsoft/applicationinsights/telemetry/JsonTelemetryDataSerializer.java

```diff
@@ -95,7 +95,9 @@ public void write(String name, int value) throws IOException {
     public void write(String name, com.microsoft.applicationinsights.internal.schemav2.SeverityLevel value) throws IOException {
         if (value != null) {
             writeName(name);
+            out.write(JSON_COMMA);
```

Comment:
Do you need a changelog item for this?

Reply:
@Dmitry-Matveev Yes we would need changelog. They are public methods so I think so yes.

Reply:
@Dmitry-Matveev @dhaval24 added changelog